api/src/main/java/io/opencensus/trace/export/SampledSpanStore.java

```diff
@@ -102,6 +103,14 @@ protected SampledSpanStore() {}
    */
   public abstract void unregisterSpanNamesForCollection(Collection<String> spanNames);
 
+  /**
+   * Returns the list of unique span names registered to the library. For this set of span names the
+   * library will collect latency based sampled spans and error based sampled spans.
+   *
+   * @return the list of unique span names registered to the library.
+   */
+  public abstract Set<String> listRegisteredSpanNamesForCollection();
```

Comment:
Should this method have a default implementation, to avoid breaking subclasses?

Reply:
nobody else implements this for the moment and I would not worry that much about other implementation, I would feel very bad if we break compatibility on instrumentations.

@dinooliva any suggestion here?